### PR TITLE
payments service

### PR DESCRIPTION
**Pull request by mgodza93**
shipment consumer
update on some entities
proper handling appointment status

Resolves #
#5 #6 #7 #9 #52 #8 

### Description
One sentence or more describing the solution that has been developed in order to resolve the ticket.

### Changes
-payments service
-shipment consumer
-update on some entities
-proper handling appointment status

### How has the solution been tested and validated
How this pull request was tested and  using waht configuration.

### Extra notes
- Attachments
- Review suggestions

### CODE REVIEW

File: backend/src/main/java/com/scc/app/service/AppointmentService.java [hunk 1]
```diff
@@ -70,4 +70,36 @@ public Appointment getAppointmentById(Long id) {
 
         return appointmentRepository.findByUserId(washingStationId);
     }
+
+    public void update(Appointment appointmentUpdated) {
```
Comment (LexuBBAA) [inline backend/src/main/java/com/scc/app/service/AppointmentService.java:74]: Shouldn't the `update` methods return the updated entries?

File: backend/src/main/java/com/scc/app/utils/Utils.java [hunk 2]
```diff
@@ -1,12 +1,23 @@
 package com.scc.app.utils;
 
 import java.util.Objects;
+import java.util.Random;
 
 public class Utils {
```
Comment (LexuBBAA) [inline backend/src/main/java/com/scc/app/utils/Utils.java:6]: Can we JUnit this up?

File: backend/src/main/java/com/scc/app/service/AppointmentService.java (hunk 1 above)
Comment (mgodza93) [reply backend/src/main/java/com/scc/app/service/AppointmentService.java:74]: fixed

File: backend/src/main/java/com/scc/app/utils/Utils.java (hunk 2 above)
Comment (mgodza93) [reply backend/src/main/java/com/scc/app/utils/Utils.java:6]: will add all the JUnit at the end

**Pull request merged**